src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const mainImage = document.querySelector('.gallery__img');
+const galleryList = document.querySelector('.gallery__list');
+
+galleryList.addEventListener('click', (e) => {
+  const link = e.target.closest('.list-item__link');
+
+  e.preventDefault();
+
+  if (!link) {
+    return;
+  }
+
+  mainImage.src = link.href;
```

Comment:
Assigning `link.href` directly to `mainImage.src` might not work as expected if `link.href` is not a valid image URL. Consider validating the URL or handling errors gracefully.